ramls/instance-iteration/iteration-job.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Iteration job",
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "Job id",
+      "type": "string",
+      "$ref": "../raml-util/schemas/uuid.schema"
+    },
+    "published": {
+      "description": "Number of records that was published so far",
+      "type": "integer",
+      "minimum": 0,
+      "default": 0
+    },
+    "jobStatus": {
+      "description": "Overall job status",
+      "type": "string",
+      "enum": [
+        "In progress",
+        "Id publishing failed",
+        "Ids published",
+        "Pending cancel",
+        "Id publishing cancelled"
+      ]
```

Comment:
One thing I learnt from using enumerations for item status is that if the enumeration values take the form of presentable text, folks tend to think they can be presented as is and forget about any need for localization.

To avoid that, I would probably use formatting that makes them obviously a constant value e.g. uppercase with underscores, rather than directly presentable text.